subxt/src/storage/utils.rs

```diff
@@ -37,3 +37,50 @@ pub(crate) fn storage_address_root_bytes<Address: StorageAddress>(addr: &Address
     write_storage_address_root_bytes(addr, &mut bytes);
     bytes
 }
+
+/// this should match the constants in the module `sp_core::storage::well_known_keys`
+pub mod well_known_keys {
```

Comment:
I'd probably keep this in `subxt::storage` and not utils (it's storage specific I think?).

Is there a use in exposing any of this stuff at all?  I'd be inclined to just inline what you need into the `runtime_wasm_code` function for now, with a pointer to where you got the key from there so we can add more if need be or whatever.

Alternately we could expose this `WellKnownKey<R>` type in `subxt::storage` and then a single `api.storage().fetch_well_known_key(key: WellKnownKey<R>) -> Result<R, Error>` type function to fetch them (it sortof depends whether any of the other keys are actually useful anywhere; that might be worth finding out!).

Reply:
This is in `subxt::storage::utils` not in the general `subxt::utils`. I wanted to expose a "nicer" interface in the hopes that many other well known keys might work in a similar fashion, such that we can abstract over their types and keys with a function like `fetch_well_known_key()` on the storage. 
But the other keys do not seem to work as intended and I think you are right, it is less bloat to just hardcode the `":code"` in the right place and remove the other stuff here. Also first I thought the returned value was scale decoded but that is also not the case apparently.

Reply:
Ok that makes sense to me; let's keep it simple then until we see a use for allowing other keys!

Reply:
It would be interesting to investigate what is causing the failure in Polkadot for the other known keys :D

Reply:
I agree!